lib/oli_web/views/authoring_view.ex

```diff
@@ -0,0 +1,51 @@
+defmodule OliWeb.AuthoringView do
+  use OliWeb, :view
+  use Phoenix.Component
+
+  alias Oli.Accounts
+  alias Oli.Accounts.{Author, User}
+
+  def author_role_text(author) do
```

Comment:
nit: Might be shorter and easier to read/scan - to just use pattern matching here

Reply:
@jrissler Agree but I was thinking that if pattern matching here, I should do it in the author user role which will make this module to depend on the system roles, and if in the future our "admin logic" change, we will need to change it here apart from `Accounts.is_admin?`. Wdyt?

Reply:
I'm thinking even simpler the if statement could even be pattern matched to be cleaner / easier to read.